### ISSUE TITLE
Compare training results between new code and old code

### ISSUE THREAD

**Issue by CunliangGeng** [mapping-gait-ai/gait-mapper#23]
Apply new code to the same data Micheal has used with old code and check if the results are consistent.

**Comment by michielpunt86** [comment]
Hi Cunliang, Not entirely sure what you are asking. Do you want me to check whether the results from your version match with the results i have obtained using my local version? If so i can just clone this project and run the code?

**Comment by CunliangGeng** [comment]
You're right, but we have not finished the code. Will give you a signal when it's done :-)

**Comment by CunliangGeng** [comment]
Hi @michielpunt86, the code is ready for testing. You could take a look at the [notebook](https://github.com/mapping-gait-ai/gait-mapper/tree/main/tutorials) about its usage. Let me or Yang know if you have questions.
